functions/composer-storage-trigger/index.js

```diff
@@ -88,6 +89,7 @@ const authorizeIap = async (clientId, projectId, userAgent) => {
   );
   const tokenResponse = await res.json();
   if (tokenResponse.error) {
+    console.error('Error in token reponse:' + tokenResponse.error.message);
```

Comment:
Tiny nit: use a comma instead of `+`? (`console.error` can take two arguments, and this is more idiomatic vs. our other samples.)